bench/commands/utils.py

```diff
@@ -17,15 +17,21 @@ def start(no_dev, concurrency, procfile, no_prefix):
 
 
 @click.command('restart', help="Restart supervisor processes or systemd units")
+@click.option('--hard', is_flag=True, default=False)
+@click.option('--soft', is_flag=True, default=False)
```

Comment:
Boolean flags might be a better fit here? Also, both defaulting to False seems a bit incorrect 🤔  The default should be current behaviour, right?

Ref: https://click.palletsprojects.com/en/7.x/options/#boolean-flags

Reply:
Thanks for the review @ankush! Actually, the issue is that the default depends on a value set in `common_site_config.json` (`soft_restart`)

Maybe we can set the default to something like `None` that gets replaced with the config value inside the function.

Reply:
We can set `default=True` for the `--hard` option imo